recipes/very-simple-smtps/all/conanfile.py

```diff
@@ -0,0 +1,171 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.apple import fix_apple_shared_install_name
+from conan.tools.build import check_min_cppstd
+from conan.tools.env import VirtualBuildEnv
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rm, rmdir, collect_libs
+from conan.tools.gnu import PkgConfigDeps
+from conan.tools.layout import basic_layout
+from conan.tools.meson import Meson, MesonToolchain
+from conan.tools.microsoft import check_min_vs, is_msvc
+from conan.tools.scm import Version
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+#
+# INFO: Please, remove all comments before pushing your PR!
+#
+
+
+class VerySimpleSmtpsConan(ConanFile):
+    name = "very-simple-smtps"
+    description = "Library that allows applications to send emails with binary attachments"
+    # Use short name only, conform to SPDX License List: https://spdx.org/licenses/
+    # In case not listed there, use "LicenseRef-<license-file-name>"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/matthewT53/Very-Simple-SMTPS/releases"
+    # no "conan" and project name in topics. Use topics from the upstream listed on GH
+    topics = ("email", "smtps", "attachments")
+    # package_type should usually be "library" (if there is shared option)
+    package_type = "library"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": False,
+    }
+
+    required_packages = [
+        "libcurl/8.0.1",
+        "doctest/2.4.11",
+    ]
+
+    @property
+    def _min_cppstd(self):
+        return 17
+
+    # in case the project requires C++14/17/20/... the minimum compiler version should be listed
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "gcc": "9",
+            "clang": "9",
+            "apple-clang": "10",
+        }
+
+    # no exports_sources attribute, but export_sources(self) method instead
+    # this allows finer grain exportation of patches per version
+    def export_sources(self):
+        export_conandata_patches(self)
+
```

Comment:
Please, remove it. There are no patches to be applied, so this part of the template has no use here.